tests/build/build_templates.go

```diff
@@ -222,16 +225,16 @@ var _ = framework.BuildSuiteDescribe("Build service E2E tests", Label("build", "
 				for i := range gatherResult {
 					if gatherResult[i] == "inspect-image" {
 						// Fetching BASE_IMAGE shouldn't fail
-						result, err := build.FetchImageTaskRunResult(pipelineRun, gatherResult[i], "BASE_IMAGE")
+						result, err := build.FetchImageTaskRunResult(f.AsKubeAdmin.CommonController.KubeRest(), pipelineRun, gatherResult[i], "BASE_IMAGE")
 						Expect(err).ShouldNot(HaveOccurred())
 						ret := build.ValidateImageTaskRunResults(gatherResult[i], result)
 						Expect(ret).Should(BeTrue())
 					} else if gatherResult[i] == "clair-scan" {
 						// Fetching HACBS_TEST_OUTPUT || TEST_OUTPUT shouldn't fail
-						result, err := build.FetchTaskRunResult(pipelineRun, gatherResult[i], constants.TektonTaskTestOutputName)
+						result, err := build.FetchTaskRunResult(f.AsKubeAdmin.CommonController.KubeRest(), pipelineRun, gatherResult[i], constants.TektonTaskTestOutputName)
```

Comment:
Based on the latest failed run's [build.log](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/redhat-appstudio_e2e-tests/486/pull-ci-redhat-appstudio-e2e-tests-main-redhat-appstudio-e2e/1652728784643690496/build-log.txt), there's a `runtime error: invalid memory address or nil pointer dereference` error (if I'm not wrong) at this line.

Reply:
I see the same error with the [periodic run ](https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/periodic-ci-redhat-appstudio-infra-deployments-main-appstudio-e2e-tests-periodic/1652825486486671360#1:build-log.txt%3A1199)